src/pages/components/pagination/style.mdx

```diff
@@ -69,3 +194,37 @@ it's being paired with.
 | Icon button      | Small (sm)  | 32 / 2          |
 |                  | Medium (md) | 40 / 2.5        |
 |                  | Large (lg)  | 48 / 3          |
+
+#### Pagination size
+
+<br />
+
+<Row>
+<Column colLg={12}>
+
+![Small, medium, and large sizes of pagination](images/pagination-usage-sizes.png)
+
+</Column>
+</Row>
+
+<Caption>Small, medium, and large sizes of pagination</Caption>
+
+#### Pagination nav size
+
+<br />
+
+<Row>
+<Column colLg={12}>
+
+![Small, medium, and large sizes of pagination nav](images/pagination-usage-sizes-pagination-nav.png)
```

Comment:
In the last 2 rows, the selected number 4 looks like it's `body-compact`?

Reply:
Visually, not sure why it looks heavier to the right and not centered. 

![image](https://github.com/user-attachments/assets/de91ab01-fd44-47b0-a97e-c09cc89332e1)